test-plan.md

```diff
@@ -93,9 +93,11 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
-#### ASP.NET Core Razor
-The Razor experience is available when you open a .cshtml file in a valid OmniSharp project. To setup a test project to verify on you can do:  
-1. `dotnet new razor`
+#### ASP.NET Core Razor Web App
+The Razor experience is available when you open a .cshtml file in a valid OmniSharp project. You should have a `dotnet` SDK >= 3.0.0-preview4 (`dotnet --version`), see https://github.com/dotnet/core-sdk for downloading newer SDKs.
```

Comment:
Once .NET Core 3.0 Preview 4 ships we can use https://aka.ms/netcore3download for acquiring the SDK. 

For testing this week I think we need to provide clearer instructions and point directly to the "Release/3.0.1XX" column on the https://github.com/dotnet/core-sdk readme page. Something like: "Install the latest preview of the .NET Core 3.0 SDK by going to https://github.com/dotnet/core-sdk and picking the appropriate platform specific installer from the Release/3.0.1XX column. The version should be 3.0.0-preview4-nnnnnn".

Reply:
+1